hoppfish/hwangsae-recorder-user-command.py

```diff
@@ -135,8 +135,56 @@ def get(self, method):
 
         return response, http_ret
 
+class vod_url(Resource):
+    def get(self, method, edge_id, file_id):
+        http_ret = 201
+        try:
+            obj = dbus.SessionBus().get_object('org.hwangsaeul.Hwangsae1.RecorderAgent',
+                    '/org/hwangsaeul/Hwangsae1/RecorderInterface')
+            dbus_interface = dbus.Interface(obj,
+                    dbus_interface='org.hwangsaeul.Hwangsae1.RecorderInterface')
+
+            ret = ''
+            if method == 'url':
+                ret = dbus_interface.Url(edge_id, file_id)
+                ret = json.dumps({'url': ret})
+
+            elif method == 'delete':
```

Comment:
I think that it is  no longer needed here because it moved to vod_delete.

Reply:
Yes, you are totally right.